### PR TITLE
9 header

### PR DESCRIPTION
**Pull request by SidletskyiOlexandr**
image
image

### CODE REVIEW

File: frontend/src/app/shared/components/header/header.component.sass [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    font-weight: 700
+    font-size: 14px
+    line-height: 146%
+
+.mat-badge-content
+    background: $time-color
+    color: $main-color
+
+p 
+  margin: 0px
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/header/header.component.sass:141]: No global styles in local css + we should have it in some PR

File: frontend/src/app/shared/components/header/header.component.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  setSocialActivePage() {
+    this.isSocialPageActive = true;
+    this.isTimetablePageActive = false;
+  }
+
+  setTimetableActivePage() {
+    this.isTimetablePageActive = true;
+    this.isSocialPageActive = false;
+  }
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/header/header.component.ts:26]: Can you check it via route?)

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

File: frontend/src/app/shared/components/header/header.component.sass (hunk 1 above)
Comment (SidletskyiOlexandr) [reply frontend/src/app/shared/components/header/header.component.sass]: removed it

File: frontend/src/app/shared/components/header/header.component.ts (hunk 2 above)
Comment (SidletskyiOlexandr) [reply frontend/src/app/shared/components/header/header.component.ts]: done

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**